ledger/accountdb.go

```diff
@@ -2596,12 +2605,7 @@ func (qs *accountsDbQueries) listCreatables(maxIdx basics.CreatableIndex, maxRes
 	return
 }
 
-type persistedValue struct {
-	value *string
-	round basics.Round
-}
-
```

Comment:
This became `persistedKVData`?  I guess I don't object too much, but it is not a Key and Value, it's only a value, which is why I called it this. I admit "persistedValue" ends up sounds pretty generic.

Reply:
Looking further into lrukv.go, it seems like we ought to keep the name you changed it to, but we ought to add `key` to it.  For accounts, `persistedAccountData` has the address AND the account data, I thing the analogy here would be for us to hold the key AND value.

Reply:
Makes sense to me, I'll modify.

Reply:
On second thought, this makes me a little nervous because I modeled the kv cache after the resource cache, which decouples the address from the persisted data object. Not sure why it works that way or if it should be changed.

Reply:
Agree! Thanks.